src/components/FiltersBar/FiltersBar.js

```diff
@@ -31,77 +31,59 @@ const movieFilters = [
   },
 ];
 
-const movieSort = [
-  {
-    id: 15,
-    isActive: true,
-    title: 'Sort by',
-  },
-  {
-    id: 16,
-    isActive: false,
-    title: 'elease Date',
-  },
-];
-
-export default class FiltersBar extends Component {
-  constructor() {
-    super();
-
-    this.defaultTitle = 'select sort';
+const InnerFiltersBar = () => {
+  const defaultTitle = 'select sort';
+  const sortList = [
+    {
+      id: 0,
+      title: 'release date',
+      selected: false,
+      key: 'sortList',
+    },
+    {
+      id: 1,
+      title: 'title',
+      selected: false,
+      key: 'sortList',
+    },
+  ];
 
-    this.state = {
-      title: this.defaultTitle,
-      sortList: [
-        {
-          id: 0,
-          title: 'release date',
-          selected: false,
-          key: 'sortList',
-        },
-        {
-          id: 1,
-          title: 'title',
-          selected: false,
-          key: 'sortList',
-        },
-      ],
-    };
+  const [title, setTitle] = useState(defaultTitle);
+  const [list, setSortList] = useState(sortList);
 
-    this.toggleSelected = this.toggleSelected.bind(this);
-  }
+  const toggleSelected = useCallback(
+    (id) => {
+      let newSortList = list;
+      let newTitile;
 
-  toggleSelected(id) {
-    let newSortList = this.state.sortList;
-    let newTitile;
-    newSortList.map((item) => {
-      if (item.id === id) {
-        item.selected = !item.selected;
-        newTitile = item.selected ? item.title : this.defaultTitle;
-      } else {
-        item.selected = false;
-      }
-    });
-    this.setState({
-      title: newTitile,
-      sortList: newSortList,
-    });
-  }
+      newSortList.map((item) => {
+        if (item.id === id) {
+          item.selected = !item.selected;
+          newTitile = item.selected ? item.title : defaultTitle;
```

Comment:
Try to stay immutable to React to avoid unexpected behavior. Ex:
`const updatedSortList = newSortList.map(item => ({ ...item, selected: item.id === id }))`
`const selectedItem = updatedSortList.find(item => item.id === id)`
`newTitle = selectedItem ? selectedItem.title : defaultTitle`

Reply:
thanks for advice. I will use it